### PR TITLE
Create: Add input project owner

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2746** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2746" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2746?utm_source=stack-comment).

**Review by tomquirk — APPROVED** [review]

File: src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx [hunk 1]
```diff
@@ -78,6 +88,34 @@ export const ProjectDetailsPage: React.FC = () => {
...
+                  label: t`Input project owner address`,
+                }),
+              ])}
+            >
+              <EthAddressInput placeholder={ADDRESS_ZERO} />
+            </Form.Item>
+            {projectOwnerDifferentThanWalletAddress && (
+              <Callout.Warning collapsible={false}>
+                <Trans>
+                  Warning: once deployed, the nominated address will become the
```
Comment (tomquirk) [inline src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx:110]: **suggesiton** can cut the `Warning` I would say

File: src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx [hunk 2]
```diff
@@ -78,6 +88,34 @@ export const ProjectDetailsPage: React.FC = () => {
           </CreateCollapse.Panel>
           <CreateCollapse.Panel
             key={1}
+            header={<OptionalHeader header={t`Project Owner`} />}
+            hideDivider
+          >
+            <Form.Item
+              className="pb-8 pt-5"
+              name="inputProjectOwner"
+              label={t`Input project owner address`}
```
Comment (tomquirk) [inline src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx:97]: Needs copy update from discord

**Review by tomquirk — APPROVED** [review]

**Comment by wraeth-eth** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: Kf6VOZ3shANW2oFFcG1d.
See details on [graphite.dev](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2746)

**Pull request merged**